mmv1/third_party/cai2hcl/common/utils.go

```diff
@@ -39,10 +36,14 @@ func DecodeJSON(data map[string]interface{}, v interface{}) error {
 
 // Converts resource from untyped map format to TF JSON.
 func MapToCtyValWithSchema(m map[string]interface{}, s map[string]*schema.Schema) (cty.Value, error) {
+	// Normalize non-marshable properties manually.
+	m = normalizeFlattenedMap(m, s).(map[string]interface{})
```

Comment:
What is this replacing? I have a vague memory of needing to do this but I don't see anything eqivalent that was removed at a quick scan.

Reply:
Oh, right, its not needed yet, because current there are no yet generated converters, this will be needed later, removed